### PR TITLE
status: Only report available when rollout is complete at new version

### PR DESCRIPTION
**Pull request by smarterclayton**
Report the release version when all deployments are updated to the latest,
to prevent the CO reporting "good" when nothing has been upgraded yet.

We are moving to a new protocol for reporting readiness to the CVO.  See https://github.com/openshift/cluster-image-registry-operator/pull/219 for more context.

Also move the autoscaler to the default run level, it's not special.

@derekwaynecarr

### CODE REVIEW

File: install/04_deployment.yaml [hunk 1]
```diff
@@ -26,6 +26,8 @@ spec:
         args:
         - -alsologtostderr
         env:
+        - name: RELEASE_NAMESPACE
```
Comment (derekwaynecarr) [inline install/04_deployment.yaml]: You mean version, not namespace.

**Comment by michaelgugino** [comment]
/hold

@smarterclayton I will pick this up, I have another patch that needs to ship first and I'll reconcile.

File: install/04_deployment.yaml (hunk 1 above)
Comment (smarterclayton) [reply install/04_deployment.yaml]: Yeah

**Comment by michaelgugino** [comment]
Closing in favor of: https://github.com/openshift/cluster-autoscaler-operator/pull/67

**Pull request closed without merging**